src/compiler/scala/tools/nsc/typechecker/RefChecks.scala

```diff
@@ -1148,7 +1148,7 @@ abstract class RefChecks extends Transform {
       val actual   = underlyingClass(other.tpe)
       def typesString = "" + normalizeAll(qual.tpe.widen) + " and " + normalizeAll(other.tpe.widen)
       def nonSensiblyEquals() = {
-        refchecksWarning(pos, s"comparing values of types $typesString using `${name.decode}` is unsafe due to cooperative equality; use `==` instead", WarningCategory.OtherNonCooperativeEquals)
+        refchecksWarning(pos, s"comparing values of types $typesString using `${name.decode}` unsafely bypasses cooperative equality; use `==` instead", WarningCategory.OtherNonCooperativeEquals)
```

Comment:
Fine by me, but I guess my ideal is `==` become the symbolic alias of `equals` and we take down the patriarchy of so-called _cooperative equality_. A system that is closed to the participation of user-land extension, like `UInt` type by Spire, directly goes against the philosophy of growable language, which Scala is named after.

Reply:
Let's seize the moment to rename the language to `Growa`. Then you can say, Grow a `UInt`. And the t-shirt can read, Grow a `Pair`.